noah/src/main/scala/edu/uci/ics/cloudberry/noah/feed/CmdLineAux.java

```diff
@@ -33,4 +35,17 @@ public static BufferedWriter createWriter(String fileName) throws IOException {
                 new OutputStreamWriter(zip, "UTF-8"));
         return bw;
     }
+
+    public static Long[] parseID(String[] stringIDs) throws CmdLineException{
```

Comment:
Great! I was using the opposite method (toPrimitive()). However, this one fits better. Done